### ISSUE TITLE
Skin second layer freaking out

### ISSUE THREAD

**Issue by DerpyDragon76** [IthanMendoza/Fresh-Moves#202]
I'm in the newest version on 1.21.3, and when i booted it up, the second layer of my skin was absolutely freaking out. the textures are pushed outwards and downwards on the arms and legs respectively, and the head texture is moving independently of the head. I'm not quite sure what the issue could be other than the new version, but that shouldn't have an affect on this
![2024-10-26_12 22 02](https://github.com/user-attachments/assets/8755d3a0-10f1-40ce-a141-e6f4b077088f)

**Comment by muzikbike** [comment]
Can confirm. Something appears to have broken with either the pack or one of the mods with the move to 1.21.2:
![2024-10-27_14 20 13](https://github.com/user-attachments/assets/aafc2b7a-5038-43e7-a5a4-55fdf567e8c4)

**Comment by hologarden** [comment]
Can confirm the same, visual error applies to skins that aren't made for the pack as well.
![2024-10-29_22 18 53](https://github.com/user-attachments/assets/995bcc22-7319-447e-80fd-a422753ae91c)

**Comment by DerpyDragon76** [comment]
> Can confirm the same, visual error applies to skins that aren't made for the pack as well. ![2024-10-29_22 18 53](https://private-user-images.githubusercontent.com/73917428/381364321-995bcc22-7319-447e-80fd-a422753ae91c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzAyNTYzNTUsIm5iZiI6MTczMDI1NjA1NSwicGF0aCI6Ii83MzkxNzQyOC8zODEzNjQzMjEtOTk1YmNjMjItNzMxOS00NDdlLTgwZmQtYTQyMjc1M2FlOTFjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMzAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDMwVDAyNDA1NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTAzMjBhODI4ZTc5NDNhZTdiOTY3ZWQ5ODIwZjRiNjc1ZDMyYjEyM2MxMGM0NmViNDU3MWZjMDE0ZDNlZjFmYzYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.teWzAdtpumAWeqBxmwCwfMgYhOmv2i1GDukc42rxGTQ)

I don't know if that's the issue, because I have the eyes still working on my skin, along with all the animations, just the skins 2nd layer is
 messed up

**Comment by OGLinsin** [comment]
Can also confirm this is broken in 1.21.3 to 1.21.1 i also have the latest version of this mod and EMF and ETF

**Comment by Pirvondal** [comment]
![image](https://github.com/user-attachments/assets/95e5865f-a525-4e0d-bb5f-319669785028)

same

**Comment by moh-id** [comment]
Yep facing the same issue on 1.21.3. Has anyone found a fix to this yet and is the issue related to ETF and EMF?

**Comment by itsnotalexthere** [comment]
> [...] one of the mods with the move to 1.21.2
It's 1.21.2+ with the changes made to all the entity models

**Comment by darksaid98** [comment]
Like mentioned. This is simply the Minecraft update _(1.21.2+)_ breaking some resource packs. 

This kind person updated the pack to 1.21.2+ [here](https://github.com/erinsc/Fresh-Moves/tree/main). Download that repository and copy the version of the pack you want into your `resourcepacks` folder.

This is only a temporary remedy until the original author/maintainer returns to release an official update to the pack.

**Comment by itsnotalexthere** [comment]
It's broken after entity model features v2 update, which makes the models
freak out

Le jeu. 12 déc. 2024 à 23:10, darksaid98 ***@***.***> a
écrit :

**Comment by wilsoffer** [comment]
<img width="1920" height="1081" alt="Image" src="https://github.com/user-attachments/assets/650f173c-1f44-4858-a15c-9606dbc3a7bb" />
Same!

**Comment by artemis8849** [comment]
Same here 

<img width="1920" height="1094" alt="Image" src="https://github.com/user-attachments/assets/7428cfb4-6c6d-4339-a456-c1d027f4e95e" />

**Comment by chip350** [comment]
its a problem with EMF, i haven't found a solution yet
for id suggest using not enough animations, even though the effects are subtle, its something.
i tried to use EMF with trailer player animations and the same thing happened to me.
i know what's causing it, basically EMF was never updated to 1.21.2+ so that happens.

**Comment by Didntknow123123** [comment]
I have second layer on head, for me helped install 3d-skin-layers mod by tr7zw
